src/components/Tutorial/__tests__/Tutorial.ts

```diff
@@ -0,0 +1,317 @@
+    await act(() => user.keyboard('{Enter}'))
+    expect(screen.getByText('You did it!')).toBeInTheDocument()
+    expect(screen.getByText('Now type something. Anything will do.')).toBeInTheDocument()
+
+    await dispatch(newThought({ value: 'my first thought' }))
+    await act(vi.runOnlyPendingTimersAsync)
+    expect(screen.getByText('Click the Next button when you are ready to continue.')).toBeInTheDocument()
+  })
+
+  describe('step second thought - create another thought', () => {
+    it('prompts to type some text to the created thought, then congratulates on typing', async () => {
+      await dispatch([
+        newThought({ value: 'already created thought' }),
+        tutorialStep({ value: TUTORIAL_STEP_SECONDTHOUGHT }),
+      ])
+      await act(vi.runOnlyPendingTimersAsync)
+      expect(screen.getByText('Try adding another thought. Do you remember how to do it?')).toBeInTheDocument()
+
+      await dispatch(newThought({ value: 'another thought' }))
+      await act(vi.runOnlyPendingTimersAsync)
+      expect(screen.getByText('Good work!')).toBeInTheDocument()
+    })
+  })
+
+  it('step subthought - how to create a thought inside thought', async () => {
+    await dispatch([
+      tutorialStep({ value: TUTORIAL_STEP_SUBTHOUGHT }),
+      newThought({
+        value: 'parent',
+      }),
+      newThought({
+        value: 'child',
+        insertNewSubthought: true,
+      }),
+    ])
+    await act(vi.runOnlyPendingTimersAsync)
+
+    expect(screen.getByText(`As you can see, the new thought "child" is nested`, { exact: false })).toBeInTheDocument()
+    expect(getByText(screen.getByTestId('tutorial-step'), 'parent', { exact: false })).toBeInTheDocument()
+  })
+
+  describe('step autoexpand - show that thoughts expand and collapse on outside click', async () => {
+    beforeAll(() => dispatch(tutorialStep({ value: TUTORIAL_STEP_AUTOEXPAND })))
+    it('tells us about thoughts being hidden when clicked away', async () => {
+      expect(
+        screen.getByText('thoughts are automatically hidden when you click away', { exact: false }),
+      ).toBeInTheDocument()
+    })
+
+    it('asks for a nested structure, then shows how child is hidden when clicked away', async () => {
+      expect(
+        screen.getByText('Oops! There are no thoughts in your thoughtspace.', { exact: false }),
+      ).toBeInTheDocument()
+
+      await dispatch(
+        importText({
+          text: `
+        - uncle
+        - parent`,
+        }),
+      )
+      await act(vi.runOnlyPendingTimersAsync)
+      expect(screen.getByText("Add a subthought and I'll show you.", { exact: false })).toBeInTheDocument()
+
+      await dispatch(newThought({ value: 'child', insertNewSubthought: true }))
+      await act(vi.runOnlyPendingTimersAsync)
+      expect(
+        screen.getByText(`Try clicking on thought "uncle" to hide subthought "child".`, { exact: false }),
+      ).toBeInTheDocument()
+
+      await dispatch(setCursorFirstMatch(['uncle']))
```

Comment:
Another example where I think it might be preferable to actually click on the thought.

Reply:
yeah, right
at first it indeed was a click, but again, due to me feeling like I'm testing the core functionality of expanding thoughts, covered already in `src/selectors/__tests__/expandThoughts.ts`, I went for a simpler approach to preserve brevity
I can of course rewrite this, but I feel like yesterday-me had a point :D